rubygem/lib/zeus/rails.rb

```diff
@@ -182,6 +182,14 @@ def test_environment
     end
 
     def test_helper
+      # don't let minitest setup another exit hook
+      begin
+        require 'minitest/unit'
+        MiniTest::Unit.class_variable_set("@@installed_at_exit", true)
```

Comment:
Is this just commentary or is there supposed to be a question in here? You say "engage you a bit" but I don't know what, if anything, you want me to respond to.